arches/management/commands/add_users.py

```diff
@@ -0,0 +1,207 @@
+"""
+ARCHES - a program developed to inventory and manage immovable cultural heritage.
+Copyright (C) 2013 J. Paul Getty Trust and World Monuments Fund
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU Affero General Public License as
+published by the Free Software Foundation, either version 3 of the
+License, or (at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+GNU Affero General Public License for more details.
+
+You should have received a copy of the GNU Affero General Public License
+along with this program. If not, see <http://www.gnu.org/licenses/>.
+"""
+
+import argparse
+import csv
+import traceback
+from django.core.management.base import BaseCommand
+from django.contrib.auth.models import User
+from django.contrib.auth.models import Group
+from django.contrib.auth.models import Permission
+from django.utils.crypto import get_random_string
+from arches.app.utils.permission_backend import assign_perm
+from arches.app.models import models
+
+
+class Command(BaseCommand):
+    """
+    Commands for adding arches test users
+
+    """
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            "-o",
+            "--operation",
+            action="store",
+            dest="operation",
+            choices=[
+                "test_users",
+                "csv_users",
+            ],
+            default="test_users",
+            help="Operation Type; "
+            + "'test_users'=Creates test users"
+            + "'csv_users'=Loads users from CSV file",
+        )
```

Comment:
Because 'test_users' as the default, it's possible to run the command like this:
```python manage.py add_users -s path/to/my.csv```

That will add three test users, but nothing from the csv, even though the csv was the intended source of the new users.

Removing this argument and just relying on the existence of the -s flag to determine the source of users might be a way to address this.